### PR TITLE
Initial cleanup

### PR DESCRIPTION
**Pull request by krystophny**

### CODE REVIEW

File: profit/main.py [hunk 1]
```diff
@@ -6,30 +6,17 @@
...
 try:
     from tqdm import tqdm
-    use_tqdm = True
-except:
-    use_tqdm = False
-
-    def tqdm(x):
-        return x
+except ModuleNotFoundError:
+    from profit.util import tqdm_surrogate as tqdm
```
Comment (krystophny) [inline profit/main.py:15]:
```
def tqdm(): return x
```

or

```
tqdm = lambda x: x
```

**Review by krystophny — COMMENTED** [review]
Please do small changes indicated

**Comment by mkendler** [comment]
Hi @krystophny!
Please review the pull request again. I also implemented some other minor things which came up during our discussion, like argparse, docstrings and other minor changes.

**Review by krystophny — COMMENTED** [review]
Very nice! I'll make one more release of the old version on Zenodo tomorrow for the current paper which still uses old confit files. Then I'll merge the PR!

**Pull request merged**